content/blog/2021-08-24-pre-defined-models-exp.md

```diff
@@ -0,0 +1,398 @@
+
+You can find the code that initializes the AlexNet model in the
+`initialize_model` function in `pretrained_model_tuner.py`. Since we have DVC
+set up, we can jump straight into fine-tuning this model to see which
+hyperparameters give us the best accuracy.
+
+We'll run the first experiment with the following command.
+
+```dvc
+$ dvc exp run
+```
+
+This will execute the `pretrained_model_tuner.py` script and run for 5 epochs
+since that's what we defined in `params.yaml`. When this finishes, you can check
+out the metrics from this run with the current hyperparameter values.
+
+```dvc
+$ dvc exp show
+```
+
+You'll see a table similar to this.
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━┳━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━┳━━━━━━┳━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━┓
+┃ neutral:**Experiment**              ┃ neutral:**Created**  ┃ metric:**step** ┃     metric:**acc** ┃    metric:**loss** ┃ metric:**training_time** ┃ param:**lr**   ┃ param:**momentum** ┃ param:**model_name** ┃ param:**num_classes** ┃ param:**batch_size** ┃ param:**num_epochs** ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━╇━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━╇━━━━━━╇━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━┩
+│ **workspace**               │ -        │    **4** │ **0.91803** │  **33.434** │        **225.35** │ **0.9**  │ **0.35**     │ **alexnet**    │ 2           │ 8          │ 5          │
+│ **main**                    │ **10:27 AM** │    **1** │ **0.88525** │ **0.28284** │        **82.143** │ **0.01** │ **0.09**     │ **squeezenet** │ **2**           │ **8**          │ **2**          │
+│ │ ╓ a9b7e7c [exp-2e1e1] │ 10:33 AM │    4 │ 0.91803 │  33.434 │        225.35 │ 0.9  │ 0.35     │ alexnet    │ 2           │ 8          │ 5          │
+│ │ ╟ 71d1e30             │ 10:32 AM │    4 │ 0.91803 │  33.434 │        225.35 │ 0.9  │ 0.35     │ alexnet    │ 2           │ 8          │ 5          │
+│ │ ╟ 18c6b09             │ 10:31 AM │    3 │ 0.92623 │  23.869 │        175.51 │ 0.9  │ 0.35     │ alexnet    │ 2           │ 8          │ 5          │
+│ │ ╟ bb2154d             │ 10:31 AM │    2 │ 0.84426 │  52.716 │        125.56 │ 0.9  │ 0.35     │ alexnet    │ 2           │ 8          │ 5          │
+│ │ ╟ 4e2f7c4             │ 10:30 AM │    1 │ 0.86066 │  67.566 │        75.783 │ 0.9  │ 0.35     │ alexnet    │ 2           │ 8          │ 5          │
+│ ├─╨ 6ad572d             │ 10:29 AM │    0 │    0.75 │  102.48 │         25.17 │ 0.9  │ 0.35     │ alexnet    │ 2           │ 8          │ 5          │
+└─────────────────────────┴──────────┴──────┴─────────┴─────────┴───────────────┴──────┴──────────┴────────────┴─────────────┴────────────┴────────────┘
+```
+
+Now let's update the hyperparameters and run another experiment. There are
+several ways to do this with DVC:
+
+- Change the hyperparameter values directly in `params.yaml`
+- Update the values using the `--set-param` option on `dvc exp run`
```

Comment:
Minor suggestion to introduce `-S` here also so that you can shorten all the commands below.